### ISSUE TITLE
On toggle NavigationController's navigation bar(show/hide) the problems with back button and its tint color

### ISSUE THREAD

**Issue by MuhammadNayabButt** [CosmicMind/Material#1182]
On previous screens I have hidden the navigation controller. When I come to the required viewcontroller I unhide/show the view controller using this command
```
self.navigationController?.setNavigationBarHidden(false, animated: true)
```

strangely after unhiding the navigation controller not even backbutton hides properly neither its tint changes.
like following two lines of code does not have any effect. I get the back button in only blue colour

```
navigationItem.backButton.isHidden = true
navigationItem.backButton.tintColor = UIColor.black
```

it seems there is a problem with native navigationController although I am using only material navigation controller.

I have attached a sample project as well which can help you debug the issue
[https://www.dropbox.com/s/wb5r7yk9hnwid1f/MaterialSample.zip?dl=0](SampleProjectLink)

**Comment by OrkhanAlikhanov** [comment]
Fixed in #1183

**Comment by danieldahan** [comment]
Thank you @OrkhanAlikhanov, https://github.com/CosmicMind/Material/pull/1183 has been merged into `development`.
